web/frontend/src/types/overrides.ts

```diff
@@ -0,0 +1,42 @@
+import type { Pose } from '@viamrobotics/sdk';
+import type { Timestamp } from 'google-protobuf/google/protobuf/timestamp_pb';
+
+export interface MappingMetadata {
+  id: string;
+  orgId: string;
+  locationId: string;
+  robotId: string;
+  timeStartSubmitted?: Timestamp;
+  timeCloudRunJobStarted?: Timestamp;
+  timeEndSubmitted?: Timestamp;
+  timeCloudRunJobEnded?: Timestamp;
+  endStatus: string;
+  cloudRunJobId: string;
+  viamServerVersion: string;
+  mapName: string;
+  slamVersion: string;
+  config: string;
+}
+
+type MappingDetails = {
+  mode: 'localize' | 'create' | 'update';
+  name?: string;
+  version?: string;
+};
+
+export interface SLAMOverrides {
+  getMappingSessionPCD: (
+    sessionId: string
+  ) => Promise<{ map: Uint8Array; pose: Pose }>;
+  startMappingSession: () => Promise<string>;
+  getActiveMappingSession: () => Promise<MappingMetadata | undefined>;
+  endMappingSession: (
+    sessionId: string
+  ) => Promise<{ packageId: string; version: string }>;
+  viewMap: (sessionId: string) => void;
+  mappingDetails: MappingDetails;
```

Comment:
Can you bring this question to the product sync today? My inclination is that it's better to avoid showing information that may change on reconfig, since the information may become inconsistent, but maybe others disagree. And it would be a good chance to advocate for https://viam.atlassian.net/browse/RSDK-3534, which is on the unordered scratchpad for SDK.

Reply:
Whoops - apologies, I didn't see the notification for this until late this afternoon. Glad you suggested it and that we'll have a complete solution to this problem!

Reply:
@sbal13 , the result from the product sync was that Fleet and SDK will schedule the work to make the RC cards reload after a reconfig, so we don't need to handle it explicitly in our PR's.